src/components/VSnackbar/VSnackbar.ts

```diff
@@ -55,17 +56,17 @@ export default {
 
   methods: {
     setTimeout () {
-      clearTimeout(this.activeTimeout)
+      window.clearTimeout(this.activeTimeout)
 
       if (this.isActive && this.timeout) {
-        this.activeTimeout = setTimeout(() => {
+        this.activeTimeout = window.setTimeout(() => {
           this.isActive = false
         }, this.timeout)
       }
     }
   },
 
-  render (h) {
+  render (h): VNode {
     const children = []
```

Comment:
`const children: VNodeArrayChildren = []`

Reply:
Do you mean `VNodeChildrenArrayContents`?

Reply:
Yeah something like that.